### DISCUSSION TITLE
[Analytical Table] Should the highlightField accessor function accept IndicationColor as a return value?

### DISCUSSION THREAD

**Discussion by Antfere** [UI5/webcomponents-react#5724] [Q&A] [answered]
Hello, in the documentation it claims that the accessor function of the highlightField prop in an Analytical Table can return a ValueState or an IndicationColor:

<img width="753" alt="image" src="https://github.com/SAP/ui5-webcomponents-react/assets/35314857/a0a1cd25-bff1-4256-84e5-0d3be3f9e758">

From my own testing this is not the case, the type definitions only accept a ValueState and at runtime IndicationColors returned by the accessor function are not functional.

I ask because I would like to use dynamic IndicationColors for my row highlights but not have to bake them into the data using the "status" key.

Thanks.

**Comment by Lukas742** [comment, marked as answer]
Hi @Antfere 
please create a [bug report](https://github.com/SAP/ui5-webcomponents-react/issues/new/choose) following our template and include an isolated example of the issue.
"Discussions" is intended as a space for the community to raise questions or share ideas, rather than for reporting issues or requesting features

**Reply by Antfere** [reply]
It turns out the code was changed two months ago to add IndicationColor, but we were still on a version from before that change. We have updated our dependencies and it works as expected, thanks.
